docs/using-the-jdbc-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,42 @@
+# Using the Limitless Connection Plugin
+
+## What is Amazon Aurora Limitless Database?
```

Comment:
Interestingly, in all of the announcements/publications of Limitless, I've only seen it referred to as "Amazon Aurora Limitless Database" singular. I think this is because "Database" is part of its name.